samples/web/content/apprtc/analytics.py

```diff
@@ -0,0 +1,91 @@
+import constants
+import datetime
+import httplib2
+import json
+import logging
+import os
+import time
+import webapp2
+
+from apiclient import discovery
+from oauth2client.appengine import AppAssertionCredentials
+from oauth2client.client import SignedJwtAssertionCredentials
+
+BIGQUERY_URL='https://www.googleapis.com/auth/bigquery'
+
+class Analytics(object):
+  """Class used to encapsulate analytics logic.
+
+  All data is streamed to BigQuery. Logging to BigQuery when running
+  locally requires a 'secrets.json' containing Service Account
+  credentials.
+
+"""
+  def __init__(self):
+    is_running_locally = os.environ['APPLICATION_ID'].startswith('dev')
+
+    self.bigquery_table = 'analytics'
+    self.bigquery_project = 'valiant-broker-834'
```

Comment:
Put this in .yaml as a variable so that another project using our code can easily change it?
(https://cloud.google.com/appengine/docs/python/config/appconfig#Python_app_yaml_Defining_environment_variables)